oracle/oracle.go

```diff
@@ -20,6 +22,9 @@ import (
 type Oracle struct {
 	VochainApp *vochain.BaseApplication
 	signer     *ethereum.SignKeys
+
+	resultQueue map[string]*models.ProcessResult
```

Comment:
As it is a temporary workaround, it is fine. Just for the record, next time you can use a simple channel for achieving the same properties of the queue, such as in vocone: https://github.com/vocdoni/vocdoni-node/blob/master/vocone/vocone.go#L52